man/org.freedesktop.sysupdate1.xml

```diff
@@ -273,6 +288,68 @@ node /org/freedesktop/sysupdate1/target/host {
       <varname>IMAGE_VERSION</varname> in <filename>/etc/os-release</filename>. If the target has no current
       version, the function will return an empty string.</para>
 
+      <para><function>ListFeatures()</function> returns a list of this target's optional features, by ID.
+      The <varname>flags</varname> argument is added for future extensibility, and must be set to 0.
+      If the target has no optional features, the method returns an empty array.</para>
+
+      <para><function>DescribeFeature()</function> returns all known information about a given optional feature.
+      The <varname>feature</varname> argument is used to pass the ID of the feature to be described.
+      The <varname>flags</varname> argument is added for future extensibility, and must be set to 0.
+      The returned JSON object contains several known keys. More keys may be added in the future.
+      The currently known keys are as follows:</para>
+
+      <variablelist>
+        <varlistentry>
+          <term><literal>name</literal></term>
+          <listitem><para>A string containing the feature's name.</para></listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><literal>description</literal></term>
+          <listitem><para>An optional string that contains a user-presentable description that identifies
+          this feature</para></listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><literal>enabled</literal></term>
+          <listitem><para>A boolean indicating whether this feature is enabled.</para></listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><literal>documentation_url</literal></term>
```

Comment:
I think we should prefer camelCase over snake_case for new JSON objects we introduce